### DISCUSSION TITLE
OSU! Lazer does not work with Nvidia GeForce Experience's "Instant Replay" Feature

### DISCUSSION THREAD

**Discussion by EARTHTOZER0** [ppy/osu#15905] [Q&A] [answered]
OSU! Lazer does not work with Nvidia GeForce Experience's "Instant Replay" Feature.

Turning it off fixes the problem but... When using Nvidia GeForce Experience's "Instant Replay" Feature, the game boots up with a black screen for a few seconds and then immediately closes itself. 
Alternatively, half of the time it will boot up with a black screen, play the boot up intro sound "Welcome to osu", play the OSU! lazer menu music for a few seconds and then close itself.

I wasn't sure where to post a bug/known issue rather than a call for help.

**Comment by gagahpangeran** [comment, marked as answer]
https://github.com/ppy/osu/issues?q=is%3Aissue+shadowplay

**Reply by frenzibyte** [reply]
Falls within https://github.com/ppy/osu/issues/10157 specifically.

**Comment by NotGumballer91** [comment]
Unsure if it's lazer's issue. It works fine on my end
